src/ListItem.js

```diff
@@ -0,0 +1,17 @@
+export default class ListItem {
+  constructor(content) { // createDate) {
+    this.id = Date.now();
```

Comment:
- [ ] Your current Todo implementation only have content and completed attribute, while as per Microverse requirements it should have (title, description, due date, priority). Let's add the missing attributes and update our applicattion accordingly.

![](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/todo_list/todo_requirement.png)